forch/config_file_watcher.py

```diff
@@ -32,67 +29,42 @@ def stop(self):
         """Stop watcher"""
         self._observer.stop()
 
-    def _reschedule_acl_file_handlers(self):
+    def schedule_acl_file_handler(self, acl_file_name, on_modified_callback):
```

Comment:
this function is unnecessarily tied to an "acl" -- what does it actually have to do with the operation of the function? it's really just "add_file_watch_handler" or something like that, correct?

Reply:
So the reason is the same with the comment above. What I wanted to do is to only store the scheduled watches for the ACL files but not faucet_structural.yaml, since the include section may change in faucet_structural.yaml.

Reply:
why can't this just be about adding a generic file watch? Why does it have to be acl files?